app/screens/apps_form/apps_form_field.tsx

```diff
@@ -49,10 +49,33 @@ const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
 });
 
 export default class AppsFormField extends React.PureComponent<Props, State> {
-    constructor(props:Props) {
+    constructor(props: Props) {
         super(props);
 
-        this.state = {};
+        let selected;
+        switch (props.field.type) {
+        case AppFieldTypes.STATIC_SELECT:
+        case AppFieldTypes.DYNAMIC_SELECT:
+        case AppFieldTypes.USER:
+        case AppFieldTypes.CHANNEL: {
+            const value = props.value as AppSelectOption[] | null;
```

Comment:
So there's no case where `value` is a single object for when it's not a multiselect? What is doing the conversion from a single object to an array? It looks like from the thread above it was decided that the App dev should not have to use an array in the single value case.

Reply:
Corrected in the latest commit.